api/server/controllers/meetingRequestController.js

```diff
@@ -0,0 +1,202 @@
+MeetingRequest = require('../models/meetingRequestModel')
+
+// Create a new meeting request
+exports.create = function(req, res) {
+    let meetingRequest = new MeetingRequest();
+    meetingRequest.author = req.params.author;
+    meetingRequest.deadline = req.params.deadline;
+    meetingRequest.groupId = req.params.groupId;
+    meetingRequest.multiPoll = req.params.multipoll ? req.params.multipoll : [];
+    meetingRequest.uniPoll = req.params.uniPoll;
+    meetingRequest.staus = req.params.status;
+    meetingRequest.isUniPoll = req.params.isUniPoll;
+    meetingRequest.totalVotes = req.params.votes;
+    meetingRequest.name = req.params.name ? req.params.name : 'Meeting';
+
+    meetingRequest.save(function(err){
+        if(err) {
+            res.json({
+                status: 500,
+                erroMessage: err.message,
+                errorName: err.name,
+            })
+        }
+        res.json({
+            status: res.statusCode,
+            message: "Meeting Request created successfully",
+            data: meetingRequest
+        })
+    })
+}
+
+// Delete a Meeting Request
+exports.delete = function(req, res) {
+    MeetingRequest.findByIdAndRemove(req.params.id , function(err, data){
+        if(err) {
+            res.json({
+                status: 500,
+                erroMessage :err.message,
+                errorName: err.name
+            })
+        } else {
+            res.json({
+                status: res.statusCode,
+                data: data
+            })
+        }
+    });    
+}
+
+// Gets all the data about the meeting request
+exports.view = function(req, res) {
+    MeetingRequest.findById(req.params.groupId, function(err, data){
```

Comment:
I think this should be finding by meetingRequestId rather than groupId? Just rename the groupId param to meetingRequestId or something equivalent (maybe just 'id' like you did above).